packages/styles/table.css

```diff
@@ -30,6 +37,43 @@
   border-bottom: 2px solid var(--table-row-border-color);
 }
 
+.TableHeader[aria-sort] {
+  padding: 0;
+}
+
+.TableHeader .TableHeader__sort-button {
```

Comment:
Not to be too OCD about it, but would it be feasible to make the border a just be around the button instead of overlapping on the sides?
![image](https://user-images.githubusercontent.com/6416791/150187635-7d521ae6-8015-49e0-b56e-0b9ce597dac8.png)

Reply:
It is overlapping on the sides because for the active column, we add a 2px border at the bottom and our universal offset for focus outline is 2px. I could manually modify the offset here.

Reply:
It is supposed to be like this.
<img width="865" alt="Screen Shot 2022-01-19 at 3 16 47 PM" src="https://user-images.githubusercontent.com/83524297/150209368-82f5b29d-fdcf-4ffa-8f51-274053555c6b.png">

Reply:
It is supposed to be like this.
<img width="814" alt="Screen Shot 2022-01-19 at 3 37 20 PM" src="https://user-images.githubusercontent.com/83524297/150209783-de9f17ac-51c5-44ab-8955-17ba432eceeb.png">